test/pummel/test-child-process-spawn-loop.js

```diff
@@ -4,26 +4,26 @@ const assert = require('assert');
 
 const spawn = require('child_process').spawn;
 
-var SIZE = 1000 * 1024;
-var N = 40;
-var finished = false;
+const SIZE = 1000 * 1024;
+const N = 40;
+let finished = false;
 
 function doSpawn(i) {
-  var child = spawn('python', ['-c', 'print ' + SIZE + ' * "C"']);
-  var count = 0;
+  const child = spawn('python', ['-c', 'print ' + SIZE + ' * "C"']);
```

Comment:
I think it's reasonable to keep the dependency here but it's also reasonable to spawn Node instead. I don't want to block this PR on it though.